.github/dependabot.yml

```diff
@@ -34,3 +34,11 @@ updates:
         update-types:
         - "minor"
         - "patch"
+      # Separate group for ASP.NET dependencies
+      aspnet:
```

Comment:
I think we want to limit these to minor and patch?

Reply:
done, thanks!